include/Graphs/VFGNode.h

```diff
@@ -55,7 +55,7 @@ class VFGNode : public GenericVFGNodeTy
     {
         Addr, Copy, Gep, Store, Load, Cmp, BinaryOp, UnaryOp, TPhi, TIntraPhi, TInterPhi,
         MPhi, MIntraPhi, MInterPhi, FRet, ARet, AParm, FParm,
-        FunRet, APIN, APOUT, FPIN, FPOUT, NPtr
+        FunRet, APIN, APOUT, FPIN, FPOUT, NPtr, DummyVProp
```

Comment:
Maybe, we can put this class just under the VSFS class instead here? Will we use it anywhere else?

Reply:
It's not used elsewhere, but `SVFG` needs to know about it. The FS worklist takes unsigned ints (IDs), and while we can technically just give it an ID that we keep track of without creating an SVFG node, we run the risk of using an integer that exists as an ID in the SVFG, causing bugs that will, probably, be terrible to debug.

Reply:
I see.